R/DBConnection.R

```diff
@@ -106,6 +106,41 @@ setGeneric("dbSendQuery",
   valueClass = "DBIResult"
 )
 
+#' Execute an SQL statement that does not produce a result set.
+#'
+#' This function should be used when you want to execute a non-
+#' \code{SELECT} query on table (ex: \code{UPDATE}, \code{DELETE},
+#' \code{INSERT INTO}, \code{DROP TABLE}, ...). It will execute
+#' the query and return the number of rows affected by the operation.
+#' (If the return value is 0 or -1, you're using it wrong.)
```

Comment:
That seems a bit strong as you might be using it to set sql options, which don't affect any rows.